core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -8368,19 +8368,50 @@ private void checkLiteral2(String expression, String expected) {
     RexNode dateRex1 = builder.call(SqlLibraryOperators.DATE,
         builder.literal("1970-02-02"));
     RexNode dateRex2 = builder.call(SqlLibraryOperators.DATE,
-        builder.literal("DATE '1970-02-02'"));
+        builder.cast(builder.literal("1970-02-02"), SqlTypeName.DATE));
```

Comment:
Why was there a need to modify inputs in existing TC?

Reply:
because it was wrong